src/Plugin/InvokeStrategy/OnEventStrategy.php

```diff
@@ -22,4 +27,22 @@ public function invoke($handler, $message): void
     {
         $handler->onEvent($message);
```

Comment:
Then I need to adjust the testcases, too and mock the handler->onEvent() call, too?